<?xml version="1.0"?>
<launch>
	<!-- Arguments -->
	<arg name="can_pose" default="1" />

	<!-- Run object positioning script -->
	<node name="set_world" pkg="arl_manipulation_baseline" type="set_world.py" output="screen" required="true" >
		<param name="can_pose" value="$(arg can_pose)"/>
	</node>

</launch>
